typescript/introduction/tsc-variable-declaration/tsc-destructuring.md

```diff
@@ -0,0 +1,116 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Destructuring
```

Comment:
This is more es5+ than TS

Reply:
Well you can say that for any JS feature.

Remember, not all people learning TypeScript will know JavaScript already.